src/SFA.DAS.Commitments.Support.SubSite/Controllers/StatusController.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Reflection;
+using System.Threading.Tasks;
+using System.Web;
+using System.Web.Http;
+using SFA.DAS.Commitments.Support.SubSite.Models;
+using SFA.DAS.Support.Shared;
+
+namespace SFA.DAS.Commitments.Support.SubSite.Controllers
+{
+    [RoutePrefix("api/status")]
+    public class StatusController : ApiController
+    {
+        // GET: Status
+        [AllowAnonymous]
+        public async Task<IHttpActionResult> Get()
```

Comment:
This line is emitting a warning: 'This async method lacks 'await' operators...'.